tempo_embeddings/text/abstractcorpus.py

```diff
@@ -212,6 +212,13 @@ def to_dataframe(
                 row["x"] = self.embeddings_2d[i, 0]
                 row["y"] = self.embeddings_2d[i, 1]
 
+            if passage.highlighting is not None:
+                row["highlight_start"] = passage.highlighting.start
+                row["highlight_end"] = passage.highlighting.end
+            else:
+                row["highlight_start"] = None
+                row["highlight_end"] = None
+
```

Comment:
This is for the DataFrame to always have those values in such column, otherwise, the rows could be "not square" (for example, if 10 passages have highlights but the other 10 don't have)